### PR TITLE
Add mozillians username in custom User model.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: marketpulse/main/views.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def list_contributions(request, user=None):
 
     contributions = Contribution.objects.filter(activity__name=FFXOS_ACTIVITY_NAME)
     if user:
-        contributions = Contribution.objects.filter(user=user, activity__name=FFXOS_ACTIVITY_NAME)
+        contributions = contributions.filter(user=user)
```
Comment (comzeradd) [inline marketpulse/main/views.py]: We don't want all contributions here. Jist the ones about Devices Prices.

Comment (akatsoulas) [inline marketpulse/main/views.py:123]: ~~Probably you need to remove this commit completely. With the gallery coming in a different url, we need to filter only the FirefoxOS activities.~~

**Comment by akatsoulas** [comment]
r+

**Pull request merged**